bin/pfcmd-old.pl

```diff
@@ -0,0 +1,2471 @@
+#!/usr/bin/perl -T
+
+=head1 NAME
+
+pfcmd - PacketFence command line interface
+
+=head1 SYNOPSIS
+
+pfcmd <command> [options]
+
+ Command:
+ checkup                     | perform a sanity checkup and report any problems
+ class                       | view violation classes
+ config                      | query, set, or get help on pf.conf configuration paramaters
+ configfiles                 | push or pull configfiles into/from database
+ floatingnetworkdeviceconfig | query/modify floating network devices configuration parameters
+ fingerprint                 | view DHCP Fingerprints
+ fixpermissions              | fix permissions of files
+ graph                       | trending graphs
+ history                     | IP/MAC history
+ ifoctetshistorymac          | accounting history
+ ifoctetshistoryswitch       | accounting history
+ ifoctetshistoryuser         | accounting history
+ import                      | bulk import of information into the database
+ interfaceconfig             | query/modify interface configuration parameters
+ ipmachistory                | IP/MAC history
+ locationhistorymac          | Switch/Port history
+ locationhistoryswitch       | Switch/Port history
+ lookup                      | node or pid lookup against local data store
+ manage                      | manage node entries
+ networkconfig               | query/modify network configuration parameters
+ node                        | node manipulation
+ nodeaccounting              | RADIUS Accounting Information
+ nodecategory                | nodecategory manipulation
+ nodeuseragent               | View User-Agent information associated to a node
+ person                      | person manipulation
+ reload                      | rebuild fingerprints without restart
+ report                      | current usage reports
+ schedule                    | Nessus scan scheduling
+ service                     | start/stop/restart and get PF daemon status
+ switchconfig                | query/modify switches.conf configuration parameters
+ switchlocation              | view switchport description and location
+ traplog                     | update traplog RRD files and graphs or obtain
+ switch IPs
+ trigger                     | view and throw triggers
+ ui                          | used by web UI to create menu hierarchies and dashboard
+ update                      | download canonical fingerprint or OUI data
+ useragent                   | view User-Agent fingerprint information
+ version                     | output version information
+ violation                   | violation manipulation
+ violationconfig             | query/modify violations.conf configuration parameters
```

Comment:
Are all these already implemented? if not it would be better to add this as they are.